cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc

```diff
@@ -609,6 +615,12 @@ Tests for the absence of a pattern predicate if an expression predicate evaluate
 |
 |
 
+| <<query-plan-undirected-all-relationships-scan,UndirectedAllRelationshipsScan>>
```

Comment:
I stole this from https://github.com/neo4j/neo4j-documentation/pull/1507, it used to be the case that this was not a necessary step but happened automatically somehow but I guess not you have to do it manually?